components/brave_wallet/browser/keyring_service.cc

```diff
@@ -1305,12 +1267,8 @@ absl::optional<std::string> KeyringService::ImportAccountForKeyring(
   if (address.empty()) {
     return absl::nullopt;
   }
-  std::vector<uint8_t> encrypted_private_key;
-  if (!encryptors_[keyring_id]->Encrypt(private_key,
-                                        GetOrCreateNonceForKeyring(keyring_id),
-                                        &encrypted_private_key)) {
-    return absl::nullopt;
-  }
+  std::vector<uint8_t> encrypted_private_key = encryptors_[keyring_id]->Encrypt(
+      private_key, GetOrCreateNonceForKeyring(keyring_id));
```

Comment:
New nonce here as well since we don't want to reuse the nonce used to encrypt the mnemonic and the imported account being added here.

Reply:
Skip this one, we'll address in a separate issue that I'll open for it.